pkg/kobject/kobject.go

```diff
@@ -92,6 +92,7 @@ type ServiceConfig struct {
 	MemLimit        yaml.MemStringorInt `compose:"mem_limit" bundle:""`
 	TmpFs           []string            `compose:"tmpfs" bundle:""`
 	Dockerfile      string              `compose:"dockerfile" bundle:""`
+	Vols            []Volumes           `compose:"" bundle:""`
```

Comment:
`Volumes` is already there  https://github.com/kubernetes-incubator/kompose/blob/master/pkg/kobject/kobject.go#L70 for libcompose mapping, `Vols` is new struct for handling volumes.